tests/tls_handler_test.c

```diff
@@ -1309,7 +1309,11 @@ AWS_STATIC_STRING_FROM_LITERAL(s_uncommon_rsa8192_host_name, "rsa8192.badssl.com
 
 static int s_tls_client_channel_negotiation_success_rsa8192_fn(struct aws_allocator *allocator, void *ctx) {
     (void)ctx;
-    return s_verify_good_host(allocator, s_uncommon_rsa8192_host_name, 443, NULL);
+    (void)s_uncommon_rsa8192_host_name;
+    (void)allocator;
+    // The certificate for rsa8192.badssl.com is expired. Disable this test until they renew the certificate or we find
+    // a better alternative. s_verify_good_host(allocator, s_uncommon_rsa8192_host_name, 443, NULL)
+    return AWS_OP_SUCCESS;
```

Comment:
https://github.com/awslabs/aws-c-io/blob/main/tests/CMakeLists.txt#L206
We disable the other test from the CmakeLists

Reply:
Thanks, updated.